airbyte-lib/airbyte_lib/_executor.py

```diff
@@ -171,18 +180,23 @@ def ensure_installation(
         venv_path = Path(venv_name)
         if not venv_path.exists():
             if not self.install_if_missing:
-                raise Exception(
-                    f"Connector {self.metadata.name} is not available - "
-                    f"venv {venv_name} does not exist"
+                raise exc.AirbyteConnectorNotFoundError(
```

Comment:
This specific branch is actionable - we can give a guidance of either install yourself or set the install_if_missing option

Reply:
Added!